rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.72.0"
+channel = "stable"
 profile = "minimal"
```

Comment:
This scares me a little bit. Do we want this to bump without us explicitly changing anything?